tests/data_access.rs

```diff
@@ -28,7 +28,7 @@ struct Loaded {
 }
 
 const TRACKABLE_MAX: u64 = 3600 * 1000 * 1000;
-const SIGFIG: u32 = 3;
+const SIGFIG: u8 = 3;
```

Comment:
Because it's capped at 5 in the data structure (e.g. largest single-unit-precision value is `2 * 10^5`). Technically it could go up to 18 before we overflow what we can do in a `u64`, but then it wouldn't be much of a `Hdr` histogram, just a regular old one. :)